h5py/tests/old/test_h5d_direct_chunk_write.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+import h5py
+import numpy
+
+from .common import ut, TestCase
+
+
+@ut.skipUnless(h5py.version.hdf5_version_tuple >= (1, 8, 11), 'Direct Chunk Writing requires HDF5 >= 1.8.11')
+class TestWriteDirectChunk(TestCase):
+    def test_write_direct_chunk(self):
+
+        filename = "direct_chunk.hdf"
+        filehandle = h5py.File(filename, "w")
```

Comment:
There should be facilities in the test suite to create and clean up temporary files (I think in the "common" module); this will dump a file into the user's current working directory.  Other tests should provide an example.